fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1613,14 +1623,32 @@ ACTOR Future<Void> postResolution(CommitBatchContext* self) {
 		                            idempotencyIdSet.param2 = kv.value;
 		                            auto& tags = pProxyCommitData->tagsForKey(kv.key);
 		                            self->toCommit.addTags(tags);
-		                            self->toCommit.writeTypedMessage(idempotencyIdSet);
+		                            if (self->pProxyCommitData->isEncryptionEnabled) {
```

Comment:
can we use `writeMutations` here? Same below.

Reply:
Ah good point yeah i think we can use that method